coredb-operator/tests/integration_tests.rs

```diff
@@ -99,6 +100,37 @@ mod test {
             "Did not find the pod {} to be ready after waiting {} seconds",
             pod_name, timeout_seconds_pod_ready
         ));
+        println!("Found pod ready: {}", pod_name);
```

Comment:
I am just putting this additional test logic into the existing test to avoid launching two coreDBs per test run, but I think there is room for improvement by setting up test fixture to allow for one, or limited set, of CoreDB to be shared between tests that would be compatible running on a shared target.

Reply:
This test addition is just checking we can run psql commands and the output looks like expected on a few basic commands.

Reply:
I think this can be accomplished with either lazy static or with rstest

Reply:
you're further along in Rust testing than I am :)

Reply:
I'll ping you if I get that part working, didn't yet